rpc-test-client/index.js

```diff
@@ -0,0 +1,67 @@
+
+let protoLoader = require('@grpc/proto-loader');
```

Comment:
These guys can be `const`.

Reply:
Unless this is a gRPC generated file, in which case skip all reviews in this file.